js/ui/control/logo_control.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+const DOM = require('../../util/dom');
+const util = require('../../util/util');
+
+/**
+ * A `LogoControl` is a control that adds the Mapbox watermark
+ * to the map as required by the [terms of service](https://www.mapbox.com/tos/) for Mapbox
+ * vector tiles and core styles.
+ *
+ * @implements {IControl}
+**/
+
+class LogoControl {
+
+    constructor() {
+        util.bindAll(['_logoRequired', '_updateLogo'], this);
```

Comment:
Nit: `_logoRequired` doesn't need to be bound (it's not used as an event handler).